internal/services/network/subnet_resource.go

```diff
@@ -148,7 +148,7 @@ func resourceSubnet() *pluginsdk.Resource {
 			"service_endpoint_policy_ids": {
 				Type:     pluginsdk.TypeSet,
 				Optional: true,
-				MinItems: 1,
+				MinItems: 0,
```

Comment:
This is essentially the same as not having it all so we should just remove it instead

Reply:
Agreed, rather than doing `service_endpoint_policy_ids = []` you can omit this value (or do `service_endpoint_policy_ids = null`) - since the API requires that if a value is specified, there's at least one item - else this has to be omitted.

Reply:
I have applied the changes